### PR TITLE
Tma 714 testing of sns lambda sqs journey

### PR DESCRIPTION
**Pull request by dthampi**
Tests for accounts

### CODE REVIEW

File: tests/event-processing-tests/run-tests.sh [hunk 1]
```diff
@@ -4,15 +4,15 @@ set -eu
...
+#cd /
 
 mkdir $TEST_ENVIRONMENT
 
 echo "Environment: $TEST_ENVIRONMENT"
 
 echo "Current Working Directory: $PWD"
 
-# gradle -q test --info --stacktrace
+ gradle -q test --info --stacktrace
```
Comment (zaydgds) [inline tests/event-processing-tests/run-tests.sh:15]: the tests were disabled as there are access control issues. as we are migrating to the self serve tool, the integration tests will also be migrated over. please leave line commented

File: tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3StepDefinitions.java [hunk 2]
```diff
@@ -56,6 +75,9 @@ public class LambdaToS3StepDefinitions {
 
     JSONObject enrichedJSON;
 
+    static String sqsoutputtext;
+    static String SqsUrl ="https://sqs.eu-west-2.amazonaws.com/750703655225/PublishToAccountsSQSQueue-build";
```
Comment (zaydgds) [inline tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3StepDefinitions.java:79]: we should not be hard coding urls as it is sensitive information. check slack for guidance

**Review by zaydgds — CHANGES_REQUESTED** [review]

File: tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3StepDefinitions.java (hunk 2 above)
Comment (dthampi) [reply tests/event-processing-tests/src/test/java/uk/gov/di/txma/eventprocessor/bdd/LambdaToS3StepDefinitions.java]: The URL is now added to runtests.sh file with a separate configuration reader file.

**Review by zaydgds — CHANGES_REQUESTED** [review]

File: tests/event-processing-tests/run-tests.sh [hunk 3]
```diff
@@ -4,15 +4,19 @@ set -eu
...
 gradle -v
 
-cd /
+#cd /
 
 mkdir $TEST_ENVIRONMENT
 
 echo "Environment: $TEST_ENVIRONMENT"
 
+export CFN_SqsURL="https://sqs.eu-west-2.amazonaws.com/750703655225/PublishToAccountsSQSQueue-build"
```
[the remark this answers is not in the document]
Comment (dthampi) [reply tests/event-processing-tests/run-tests.sh]: This is now updated to include in the ssm

**Pull request closed without merging**